src/status_im/contexts/wallet/wallet_connect/core.cljs

```diff
@@ -11,6 +13,13 @@
   [chain-id]
   (str "eip155:" chain-id))
 
+(defn eip155->chain-id
+  [chain-id-str]
+  (-> chain-id-str
+      (string/split #":")
+      (last)
+      (edn/read-string)))
+
```

Comment:
Can be 
```clj
(-> chain-id-str
      (string/split #":")
      last
      edn/read-string)
```

Methods without args need not be wrapped in `()`

Reply:
I know, I just always do it because it looks nicer.